h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -222,6 +222,13 @@ public static Route registerEndpoint(String method_uri, Class<? extends RestApiH
   @Override protected void doPost(HttpServletRequest rq, HttpServletResponse rs)   { doGeneric("POST", rq, rs); }
   @Override protected void doHead(HttpServletRequest rq, HttpServletResponse rs)   { doGeneric("HEAD", rq, rs); }
   @Override protected void doDelete(HttpServletRequest rq, HttpServletResponse rs) { doGeneric("DELETE", rq, rs); }
+  @Override protected void doOptions(HttpServletRequest rq, HttpServletResponse rs) {
+    if (System.getProperty(H2O.OptArgs.SYSTEM_PROP_PREFIX + "debug.cors") != null) {
```

Comment:
Minor: DRY - you can use `private static String SYS_PROP_DEBUG_CORS = H2O.OptArgs.SYSTEM_PROP_PREFIX + "debug.cors"`